CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 
 * Added a configuration option under `docker.hash-lookup.enabled` to disable docker hash lookup.
  Disabling it will also disable call caching for jobs with floating docker tags.
+ 
+* **API**    
+    + Updated the `/query` response to include the total number of query results returned. See [here](http://cromwell.readthedocs.io/en/develop/api/RESTAPI/#workflowqueryresponse) for more information.
+
+## 30.1 Release Notes
+
+* A set of bug fixes following the migration of Cromwell to WOM (the Workflow Object Model) in version 30.
```

Comment:
ToL: weird. not sure how the 30.1 change notes are just now being added.

Reply:
I remember noticing that they weren't on there when it was released and since I could make an argument either way I didn't say anything.

OTOH this would be for a `30.2` release as `30.1` is already out ....

Reply:
I just didn't like that they were missing.....but I can take them out of this PR and add them separately?

Reply:
I like them (although it's a little strange to see them in this PR, they're better here than nowhere!)